src/CMakeLists.txt

```diff
@@ -438,6 +438,7 @@ set(common_SRCS
 	server.cpp
 	serverenvironment.cpp
 	serverlist.cpp
+	serveractiveobjectmap.cpp
```

Comment:
Needs to be added to https://github.com/minetest/minetest/blob/master/build/android/jni/Android.mk as well.

Reply:
Done.
BTW, why do you write URL and not the relative path?

Reply:
@numberZero Mostly lazyness, as I've already bookmarked that page, so it shows up on top when looking up "Android" :100: